Eevee.Sleep.Bot/Controllers/Mongo/MongoIndexManager.cs

```diff
@@ -10,7 +10,8 @@ public static class MongoIndexManager {
             ActivationDataSourceIndex(),
             ActivationPresetSourceIndex(),
             ActivationPresetTagIndex(),
-            ActivationPresetUuidIndex()
+            ActivationPresetUuidIndex(),
+            RoleUserIdIndex()
```

Comment:
Since DB name changed, change the name to `DiscordRoleRecordIndex()` (I would call what we're doing here a "role record"`

Reply:
Fixed!
[9a44466](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/9a44466615d65b9af287798a6c556c2aad368643)